### DISCUSSION TITLE
What can I define in the SimpleSchema option `uniform`?

### DISCUSSION THREAD

**Discussion by rafaeltuelho** [vazco/uniforms#947] [Q&A] [answered]
I see this "custom" `uniforms` option in the example following `SimpleSchema` snippet:

```
    rating: {
      type: Number,
      allowedValues: [1, 2, 3, 4, 5],
      uniforms: { checkboxes: true }, // <<< HERE
    },

```

could not find documentation for this? Can you please point me to the docs or src code to learn what can I use here?

**Comment by radekmie** [comment, marked as answer]
Hi @rafaeltuelho. As described in [the docs](https://uniforms.tools/docs/api-bridges#simpleschema2bridge), you can pass **any** props in there.

**Reply by rafaeltuelho** [reply]
Got it! 
From the example in the docs is not clear that I can refer to/use the Uniforms component's properties. Maybe a small note could make it more clear.
